src/lib/Scenes/City/cityTabs.ts

```diff
@@ -0,0 +1,34 @@
+import { MapTab } from "../Map/types"
+
+export const cityTabs: MapTab[] = [
+  {
+    id: "all",
+    text: "All",
+    getShows: bucket => bucket.galleries,
+    getFairs: bucket => bucket.fairs,
+  },
+  {
+    id: "saved",
+    text: "Saved",
+    getShows: bucket => bucket.saved,
+    getFairs: _ => [],
+  },
+  {
+    id: "fairs",
+    text: "Fairs",
+    getShows: _ => [],
+    getFairs: bucket => bucket.fairs,
+  },
+  {
+    id: "galleries",
+    text: "Galleries",
+    getShows: bucket => bucket.galleries,
+    getFairs: _ => [],
+  },
+  {
+    id: "museums",
+    text: "Museums",
+    getShows: bucket => bucket.museums,
+    getFairs: _ => [],
+  },
+]
```

Comment:
maybe these could be renamed to clarify they're on the map? Unsure, not a blocker for me though